### PR TITLE
Add test for certificate generation

### PR DESCRIPTION
**Pull request by periklis**
This PR extends the unit test coverage for the entire CLO certificate generation.

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -186,6 +187,14 @@ func GetShareDir() string {
 	return shareDir
 }
 
+func GetScriptsDir() string {
+	scriptsDir := os.Getenv("SCRIPTS_DIR")
```
Comment (ewolinetz) [inline pkg/utils/utils.go:191]: i don't think there's a need/want to let users override this value

**Comment by ewolinetz** [comment]
holding to review -- the CLO never generated certificates for metrics, it was auto generated because if we create/sign it ourselves then it will not be trusted (using our own self signed certs)

**Comment by ewolinetz** [comment]
per the bz - they are using the incorrect port:
```
Get https://10.128.2.51:60000/_prometheus/metrics: x509: certificate is valid for localhost, elasticsearch, elasticsearch.cluster.local, elasticsearch.openshift-logging.svc, elasticsearch.openshift-logging.svc.cluster.local, not elasticsearch-metrics.openshift-logging.svc
```

per EO:
```
			// HTTPs listener for metrics
			"--metrics-listening-address=:60001",
			"--metrics-tls-cert=/etc/proxy/secrets/tls.crt",
			"--metrics-tls-key=/etc/proxy/secrets/tls.key",
```

File: pkg/utils/utils.go (hunk 1 above)
Comment (jcantrill) [reply pkg/utils/utils.go:191]: this is inline with our current pattern for workingdir... user's could only override if they go unmanaged, edit the deployment, kick the pod

Comment (ewolinetz) [reply pkg/utils/utils.go:191]: i still think its unnecessary to provide this, it was never configurable before and not sure what it actually provides...

Comment (periklis) [reply pkg/utils/utils.go:191]: The issue here is that method under test `CreateOrUpdateCertificates` relied on a hardcoded string. However, we need to be able to setup a proper dir for unit and e2e tests independent of container fs layout.

**Pull request merged**